wiki/Tournaments/GBC/GBC_2022_Spring/en.md

```diff
@@ -0,0 +1,208 @@
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | profile badge, KanonBot profile badge, 50 CNY ￥ per team member |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | KanonBot profile badge, 30 CNY ￥ per team member |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | KanonBot profile badge, 20 CNY ￥ per team member |
+
+## Organisation
+
+The GB Cup 2022 Spring was run by the GB Team and various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) |
+| Staff | ![][flag_CN] [\[GB\]AelSan](https://osu.ppy.sh/users/14095291),![][flag_CN] [IA Daisuki](https://osu.ppy.sh/users/11238501),![][flag_GB] [-Deepdive-](https://osu.ppy.sh/users/13338645) |
+| Mappool selector | ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14095291),![][flag_CN] [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561),![][flag_CN] [\[GB\]Yuria](https://osu.ppy.sh/users/13092888),![][flag_CN] [IA Daisuki](https://osu.ppy.sh/users/11238501) |
+| Mappool tester | ![][flag_CN] [\[GB\]Thaumiel](https://osu.ppy.sh/users/15017771),![][flag_CN] [\[GB\]Akamite](https://osu.ppy.sh/users/13418334),![][flag_CN] [Blue_Potion](https://osu.ppy.sh/users/13094831),![][flag_CN] [FleVI](https://osu.ppy.sh/users/12519616),![][flag_CN] [Takeru3](https://osu.ppy.sh/users/15188014),![][flag_CN] [_Yiiiiii](https://osu.ppy.sh/users/6066359),![][flag_CN] [rbqq](https://osu.ppy.sh/users/13747488/mania),![][flag_CN] [Seven_Nye](https://osu.ppy.sh/users/20886966),![][flag_CN] [aa219040](https://osu.ppy.sh/users/11653961),![][flag_CN] [DongDouFu](https://osu.ppy.sh/users/17621613) |
+| Referee | *TBA* |
+| Commentator | ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) ![][flag_CN] [aiyulu](https://osu.ppy.sh/users/189617) ![][flag_CN] [\[GB\]AelSan](https://osu.ppy.sh/users/14095291) ![][flag_CN] [Edward_Tsui](https://osu.ppy.sh/users/8609627) |
+| Designer | *TBA* |
+| Statistician | ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407)，![][flag_CN] [Rush_FTK](https://osu.ppy.sh/users/3046856)  |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1546120)
+- Livestream
+- Challonge bracket
+- Master sheet
+- Statistics sheet
+
+## Ruleset
+
+### Tournament rules
+
+1. The GB Cup 2022 Spring is a captain draft-based team tournament, played on the osu!mania game mode.
+   - **While this competition is planned as a 3 versus 3 setup, this might change depending on the number of incoming registrations.**
+2. Beatmap scoring will be based on Score V2.
+3. The beatmaps for each round will be announced by the Mappool Selector Team on the Sunday before the actual matches take place.
+   - Each mappool will contain a *tiebreaker* beatmap. It will only be played in case of a tie in the scoreline in the *best of* system, e.g. the scoreline is 4-4 and the match is best of 9, the tiebreaker shall be played.
```

Comment:
```suggestion
   - Each mappool will contain a *tiebreaker* beatmap. It will only be played in case of a tie in the scoreline in the *best of* system, e.g. if the scoreline is 4-4 and the match is best of 9, the tiebreaker shall be played.
```

Reply:
this rule's description is a copy from MWC 4k 2021, so I thought it might right.